connectors/sources/salesforce.py

```diff
@@ -1340,10 +1352,26 @@ def get_default_configuration(cls):
                 "tooltip": "The client secret for your OAuth2-enabled connected app. Also called 'consumer secret'",
                 "type": "str",
             },
+            "standard_objects_to_sync": {
+                "display": "textarea",
+                "label": "Standard Objects to Sync",
+                "order": 4,
+                "tooltip": "The list of Salesforce Standard Objects to sync",
```

Comment:
We should explain wildcard somewhere

```suggestion
                "tooltip": "The list of Salesforce Standard Objects to sync, use '*' to include all",
```